tick/base/array/tests/src/array_gtest.cpp

```diff
@@ -520,6 +520,28 @@ TYPED_TEST(ArrayTest, SortAbsTrackDecreasing) {
 namespace {
 
 template <typename ArchiveIn, typename ArchiveOut, typename TypeParam>
+void TestEmptySerialization() {
```

Comment:
This test is nothing but `TestSerialization` for an empty array with tests on expected size.